notebooks/notebooks/iclr2022/estimate-flops-katago.ipynb

```diff
@@ -0,0 +1,662 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #7.        df_nets.query("rows &lt;= 2898845681 &amp; net_size != 'b60c320'")
</pre><p>Unclear to reader where this number comes from. Can you add it to a variable to make it less magical, like <code>strongest_confidently_rated_network = 2898845681</code>?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/HumanCompatibleAI/KataGoVisualizer/pull/40/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>

Reply:
Added variable and removed magic number.